src/sortedtable/sortedtable.tsx

```diff
@@ -82,6 +92,7 @@ interface SortedTableProps<T> {
   // empty state for table
   empty?: boolean;
```

Comment:
Do you think we need `empty` prop so far? It looks like, with current changes, it's possible to calculate if there is some data to display or not.

Reply:
In admin-ui this prop is used heavily, I would leave as is at least for this iteration of refactoring.